### DISCUSSION TITLE
How does maxpendingchannels work?

### DISCUSSION THREAD

**Discussion by LNOrville** [lightningnetwork/lnd#7257] [Q&A] [answered]
At this time I am not interested in receiving new channels to my node. I thought about setting --maxpendingchannels=0 but would that setting prevent me from opening new channels as well?

**Comment by yyforyongyu** [comment, marked as answer]
> I thought about setting --maxpendingchannels=0 but would that setting prevent me from opening new channels as well?

It only applies to incoming channels, so you are free to open channels.
